Scenarioo/ScenariooTest/ScenariooTest/ScenarioDocuWriterTest.cs

```diff
@@ -34,147 +35,153 @@
 
 namespace ScenariooTest
 {
-    [TestClass]
+    [TestFixture]
     public class ScenarioDocuWriterTest
     {
-        private const string BranchName = "testBranch";
-
+        private const string BranchName = "csharp-writer-unit-tests";
         private const string BuildName = "testBuild";
-
-        private const string UseCaseName = "testCase";
-
+        private const string SerializationUseCase = "Serialization";
         private const string ScenarioName = "testScenario";
-
         private const string DetailsVersionKey = "version";
-
         private const int StepIndex = 1;
         
         private string rootDirectory;
 
         private ScenarioDocuWriter writer;
-
         private ScenarioDocuReader reader;
-
         private ScenarioDocuFiles docuFiles;
 
-        [TestInitialize]
+        [TestFixtureSetUp]
         public void TestInit()
         {
             // Sets outcome directory
-            this.rootDirectory = Path.Combine(Directory.GetCurrentDirectory(), "testoutcome");
-
-            if (!Directory.Exists(this.rootDirectory))
+            rootDirectory = Path.Combine(Directory.GetCurrentDirectory(), "testoutcome");
+            TestCleanUp();
+            
+            if (!Directory.Exists(rootDirectory))
             {
-                Directory.CreateDirectory(this.rootDirectory);
+                Directory.CreateDirectory(rootDirectory);
             }
 
-            this.writer = new ScenarioDocuWriter(
-                this.rootDirectory,
+            writer = new ScenarioDocuWriter(
+                rootDirectory,
                 BranchName,
                 BuildName);
 
-            this.reader = new ScenarioDocuReader(this.rootDirectory);
+            reader = new ScenarioDocuReader(rootDirectory);
 
-            this.docuFiles = new ScenarioDocuFiles(this.rootDirectory);
+            docuFiles = new ScenarioDocuFiles(rootDirectory);
         }
 
-        [TestCleanup]
+        //[TestFixtureTearDown]
```

Comment:
@tobiaszuercher Can you clean this up, please? I think the [TestFixtureTeardown] is not needed anymore, instead this method should be named "cleanOutputDirectory()" which is now called on init. correct?

Another question: What is the ScenarioDocuReader? I thought the C# library does not support reading?